rbd/rbd_test.go

```diff
@@ -131,6 +131,11 @@ func TestImageReadOnly(t *testing.T) {
 	image, err := rbd.Create(ioctx, name, 1<<22, 22)
 	assert.NoError(t, err)
 
+	// an integer is not a valid argument
+	err = image.Open(123)
```

Comment:
*sigh* this is stuff that the type system should be preventing. ah well, this is better than nothing.